scripts/src/main/resources/scripts/command/ide

```diff
@@ -42,7 +42,13 @@ function doUpdateSettings() {
         echo "Please contact the technical lead of your project to get the SETTINGS_URL for your project."
         echo "In case you just want to test devonfw-ide you may simply hit return to install default settings."
         echo
-        read -r -p "Settings URL: " answer
+        if [ "${DEFAULT_SETTING_URL}" == "default" ]
+        then
+          echo  "Settings URL:"  "${DEFAULT_SETTING_URL}"
+          SETTINGS_URL="-"
+        else
+          read -r -p "Settings URL: " answer
```

Comment:
As we know when control comes on the line 50  'read -r -p "Settings URL: " answer' its need manual setting_url but in the case of automation we can't put manual anything that's why i am trying to bypass this line with the lines of code which i have added.
Parameter 'DEFAULT_SETTING_URL' and its Value 'default' i am sending from other bat file which have added through java( swtbot automation java part). Line 50 only will be bypass in case of swtbot automation ,In other case work as it is.

Reply:
Can we do like this -
echo
        if [ "${answer}" == "-" ]
        then
          SETTINGS_URL="-"
        else
          read -r -p "Settings URL: " answer
        fi
instead of above code ,is it ok?

Reply:
There is nothing you need to change at all. It is all already working.
Just call setup with "-" as parameter as I already stated.
Could you read my comment in the issue and follow the link to the existing integration test.
Wenn discussion started about swtbot for devonfw-ide I pointed out in daily standup that people should first have a look at the existing integration tests. It would be lovely if you could first have a look at this so we can avoid problems that are already solved. Thanks.